packages/core/src/EventManager.ts

```diff
@@ -25,7 +25,7 @@ export class EventManager {
     /**
      * Callback given to the normalizer.
      */
-    _onNormalizedEvent(customEvent: CustomEvent): void {
+    async _onNormalizedEvent(customEvent: CustomEvent): Promise<void> {
```

Comment:
You put `async` and `Promise` but no `await`, how does that work ? I see most of the time we `return this.editor.execCommand` so I guess it works, but the `default` case does not `return`.

Reply:
We return an asynchronous function call so that checks out. The default case I'm not sure...